pkg/contract/store.go

```diff
@@ -27,6 +27,15 @@ var StandardContractTemplates = TemplateStore{
 			TemplateAttributes: []string{ActingPartyAttr, LegalEntityAttr, ValidFromAttr, ValidToAttr},
 			Regexp:             `NL:BehandelaarLogin:v2 Ondergetekende geeft toestemming aan (.+) om namens (.+) en ondergetekende het Nuts netwerk te bevragen. Deze toestemming is geldig van (.+) tot (.+).`,
 		},
+		"v3": &Template{
+			Type:               "BehandelaarLogin",
+			Version:            "v2",
+			Language:           "NL",
+			SignerAttributes:   StandardSignerAttributes,
+			Template:           `NL:BehandelaarLogin:v3 Hierbij verklaar ik te handelen in naam van {{` + LegalEntityAttr + `}}. Deze verklaring is geldig van {{` + ValidFromAttr + `}} tot {{` + ValidToAttr + `}}.`,
+			TemplateAttributes: []string{LegalEntityAttr, ValidFromAttr, ValidToAttr},
+			Regexp:             `NL:BehandelaarLogin:v3 Hierbij verklaar ik te handelen in naam van (.+). Deze verklaring is geldig van (.+) tot (.+).`,
```

Comment:
no care organizations with a dot in the name?

Reply:
a trailing dot you mean

Reply:
dots are part of sentences not names. Let's add a requirement to the organisation proof/registry entry

Reply:
I'll add it to RFC006 (permitted characters)